md-setup/src/main/java/com/wipro/ats/bdre/md/setup/SetupDB.java

```diff
@@ -143,9 +142,11 @@ private void populateBatch(String dataFile) throws Exception {
             BufferedReader br = new BufferedReader(new FileReader(dataFile));
             while ((line = br.readLine()) != null) {
                 lineNum++;
-                LOGGER.debug("Line #" + lineNum + ": " + line);
+                String constant="Line #";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Line #" 16 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)